examples/dhs-example/README.md

```diff
@@ -0,0 +1,97 @@
+    
+## Deploying resources as a data-hub-developer user
+
+As of 5.2.0, a user with the data-hub-developer is permitted to deploy the following resources:
+
+- Indexes to the data-hub-STAGING, data-hub-FINAL, and data-hub-JOBS databases
+- Artifacts (entities, flows, mappings, and step definitions)
+- Modules
+- Alert configs, rules, and actions
+- Scheduled tasks
+- Schemas
+- Temporal axes and collections
+- Triggers
+
+Permitted resources can be deployed via the following task (assuming that gradle-dhs.properties defines the host and
+credentials for the DHS instance):
+
+    ./gradlew -PenvironmentName=dhs dhsDeployAsDeveloper
+    
+Each of the permitted resources is described below in more detail.
+
+### Database indexes
+
+Each database file that references data-hub-STAGING, data-hub-FINAL, and data-hub-JOBS will be processed. Database files
+are located in the "./databases" directory in a configuration directory. In this example project, the "test-database.json"
+file will not be processed since it does not reference one of those 3 databases. Also see 
+[the ml-gradle docs](https://github.com/marklogic-community/ml-gradle/wiki/Resource-reference#databases) for more information.
+
+## Artifacts
+
+Equivalent to on-premise deployment, all Data Hub artifacts in the ./entities, ./flows, ./mappings, and 
+./step-definitions directories will be deployed.
+
+## Modules
+
+All user modules in any directory specified by mlModulePaths (defaults to src/main/ml-modules) will be deployed. Contrary
+to an on-premise deployment, the Data Hub modules themselves are not deployed. These are instead deployed automatically
+within the DHS instance. Also see [the ml-gradle docs](https://github.com/marklogic-community/ml-gradle/wiki/How-modules-are-loaded) for more information. 
+
+## Alert resources
+
+Alert resource files - configs, alerts, and rules - are stored in the directory "./databases/(name of content database)/alerts". 
+See src/main/ml-config/databases/data-hub-FINAL/alert for an example in this project. Also see 
+[the ml-gradle docs](https://github.com/marklogic-community/ml-gradle/wiki/Resource-reference#alerting) for more information.
+
+## Schemas 
+
+Schema files are stored in the directory "./databases/(name of schema database)/schemas". See 
+src/main/ml-config/databases/data-hub-staging-SCHEMAS/schemas for an example in this project. Also see 
+[the ml-gradle docs](https://github.com/marklogic-community/ml-gradle/wiki/Loading-schemas) for more information. 
+
+## Scheduled tasks
+
+Because a DHS instance uses multiple groups for application servers, and because a scheduled task is associated with a
+group, scheduled task files are stored in the directory "./tasks/(name of group)". See 
+src/main/ml-config/tasks for an example in this project. Also see
+[the ml-gradle docs](https://github.com/marklogic-community/ml-gradle/wiki/Resource-reference#scheduled-tasks) for more information.
+
+## Temporal axes and collections
+
```

Comment:
Is it worth mentioning here that lsqt is not supported?

Reply:
That's a good idea,  I added mention of that.